client/components/groupevents/userModal.jsx

```diff
@@ -0,0 +1,205 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import { PropTypes } from 'prop-types';
+import axios from '../../utils/setAuthToken';
```

Comment:
I really question this o. You save a file as setAuthToken and you are importing with a name axios. If it were to be destructuring I think is Ok. but importing a default export like this, I doubt if is a good practice